app/upw/controllers/taskList.utils.js

```diff
@@ -108,7 +108,7 @@ const getTaskList = (baseUrl = '', steps = {}, answers = {}) => {
       })
     }, tasks)
 
-    if (taskIndex && itemIndex) delete tasks[taskIndex].items[itemIndex]
+    if (taskIndex && itemIndex) delete tasks[taskIndex].items.splice(itemIndex, 1)
```

Comment:
Noticed this bug whilst testing, fixed to avoid having empty items on the task list